index.html

```diff
@@ -8,6 +8,7 @@
             rel="stylesheet"
             href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
         />
+        <link rel="icon" href="img/portfolio_icon.png" />
```

Comment:
- [ ] - [ OPTIONAL] Am taking reference from the HTML file to raise some optional suggestions I found you can try to add when I was checking your design, there are some work cards that don't have images, not because they are broken but because you did not provide any for them, I think it may be fine if you add images to all of your projects as you did with some.
- [ ] - [ OPTIONAL ] The next thing is your header in the desktop version, when the user scrolls, the text are getting confused in the body, I think you can add a white background to it or remove the fixed position to allow it scrolling accordingly with the body.
- [ ]  - [ OPTIONAL ] Still in the desktop version, your `about me` technology cards are a bit not well centered when they have an `odd` total number {3,5,7....}, You will see in the record below, I think you can justify the content of the parent container to `center` so the last card will be aligned in the middle too.


![5](https://user-images.githubusercontent.com/77843050/207104092-1ac2e526-c876-464f-9489-f288f7154d70.gif)